src/index.html

```diff
@@ -10,13 +10,104 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
```

Comment:
The attributes of the `meta` tag should start on a new line with a 2-space indentation related to the tag since there are more than two attributes.